web/modules/custom/modal_form_example/src/Form/Demo.php

```diff
@@ -0,0 +1,314 @@
+      ];
+
+      $form['current'] = [
+        '#type' => 'markup',
+        '#markup' =>
+        '<div class = "profilefield"><img class="picimage" src = "/sites/default/files/styles/post_thumbnail_30_x_30_/public/pictures/2022-01/' .
+        $picture .
+        '">
+                    <div class = "profname">' .
+        $name .
+        '</div></div>',
+        '#prefix' => '<div class = "data">',
+      ];
+      $form['body'] = [
+        '#type' => 'textarea',
+        '#required' => TRUE,
+        '#default_value' => $form_state->getValue('body', ''),
+        '#attributes' => [
+          'class' => ['textbody'],
+        ],
+        '#placeholder' => 'Write a caption...',
+      ];
+      $form['accesstitle'] = [
+        '#type' => 'markup',
+        '#markup' => '<div class = "access-title" id ="accesstitle">
+                Accessibility
+                <i class="fas fa-angle-down"></i>
+                </div>',
+      ];
+      $form['accessible'] = [
+        '#type' => 'textfield',
+        '#prefix' =>
+        '<div class = "accessibility" id ="accessibility">
+                    <div class = "access1" id="access1">
+                    <h2>Accessibility</h2>
+                   <i class="fas fa-angle-up"></i>
+                    </div>
+                    <p>Alt text describes your photos for people with visual impairments. Alt text will be automatically created for your photos or you can choose to write your own.</p>
+                    <div class = "alttext"><img src = "/sites/default/files/' .
+        $uri .
+        '"> ',
+        '#placeholder' => ' Write alt Text',
+        '#suffix' => '</div>',
+        '#suffix' => '</div>',
+        '#suffix' => '</div>',
+
+      ];
+    }
+
+    if ($this->step == 1) {
+      $form['buttons']['forward'] = [
+        '#type' => 'submit',
+        '#value' => $this->t('Next'),
+
+        '#attributes' => [
+          'class' => ['nextbutton'],
+        ],
+        '#ajax' => [
+                // We pass in the wrapper we created at the start of the form.
+          'wrapper' => 'ajax_form_multistep_form',
+                // We pass a callback function we will use later
+                // to render the form for the user.
+          'callback' => '::ajaxFormMultistepFormAjaxCallback',
+          'event' => 'click',
+        ],
+      ];
+    }
+    if ($this->step == 2) {
+      $form['buttons']['forward'] = [
+        '#type' => 'submit',
+        '#prefix' => '</div>',
+        '#value' => $this->t('Share'),
+        '#attributes' => [
+          'class' => ['submitbutton'],
+        ],
+        '#ajax' => [
+                // We pass in the wrapper we created at the start of the form.
+          'wrapper' => 'ajax_form_multistep_form',
+                // We pass a callback function we will
+                // use later to render the form for the user.
+          'callback' => '::ajaxFormMultistepFormAjaxCallback',
+          'event' => 'click',
+        ],
+      ];
```

Comment:
Add the common code all together and if conditions for changing the specific parts only, avoid code duplication